playbooks/roles/ses-rook/defaults/main.yml

```diff
@@ -1,5 +1,5 @@
 ses_rook_repos:
-  ses_rook_sle15_sp1: "http://download.suse.de/ibs/Devel:/Storage:/6.0/SLE_15_SP1/"
+  ses_rook_sle15_sp1: "http://download.suse.de/ibs/SUSE:/SLE-15-SP1:/Update:/Products:/SES6:/Update/standard/"
```

Comment:
we probably want to use the provo mirror (when it works again): https://github.com/SUSE-Cloud/socok8s/pull/744/files#diff-0d7e3c79df5eb0ccb93627cfaa14b956L23

Reply:
Updated to use the variable, please check again